src/src/presentation/access/AccessNavigator.tsx

```diff
@@ -21,7 +21,12 @@ import { SignupPhoneVerifiedScreen } from "./signup/SignupPhoneVerified";
 import { SignupVerifyPhoneScreen } from "./signup/SignupVerifyPhone";
 import { SignupWithResetScreen } from "./signup/SignupWithReset";
 import { StartAccessScreen } from "./StartAccess";
-
+import { CommonQuestionsScreen } from "../common/CommonQuestions";
+import { PoliticsScreen } from "../common/Politics";
+import { OpenEmailScreen } from "../common/OpenEmail";
+import { ExpiredAccountScreen  } from "../common/ExpiredAccount";
```

Comment:
verificar y corregir sobre este archivo , marca este error al levantar la aplicación : 
Error: Unable to resolve module

Reply:
lo reviso

Reply:
Listo @farpires